trailblazer/store/api.py

```diff
@@ -257,19 +257,19 @@ def set_analysis_completed(self, analysis_id: int) -> None:
 
     def set_analysis_uploaded(self, case_id: str, uploaded_at: dt.datetime) -> None:
         """Setting analysis uploaded at."""
-        analysis_obj: Analysis = self.get_latest_analysis(case_id=case_id)
+        analysis_obj: models.Analysis = self.get_latest_analysis(case_id=case_id)
```

Comment:
Do we want to use these types of imports?

Reply:
Fixed here: https://github.com/Clinical-Genomics/trailblazer/pull/209